zipline/algorithm.py

```diff
@@ -2367,6 +2375,12 @@ def default_pipeline_domain(calendar):
         """
         return _DEFAULT_DOMAINS.get(calendar.name, domain.GENERIC)
 
+    @staticmethod
+    def default_fetch_csv_country_code(calendar):
+        """
+        """
+        return _DEFAULT_FETCH_CSV_COUNTRY_CODES.get(calendar.name)
```

Comment:
Why does this need to be a `get` instead of normal key access? Do we ever expect this to fail, and if so what happens when it returns `None`?

Reply:
Thanks for the explanation, I think what you have is good then.